torchgeo/datasets/fair1m.py

```diff
@@ -28,21 +28,25 @@ def parse_pascal_voc(path: str) -> Dict[str, Any]:
     Returns:
         dict of image filename, points, and class labels
     """
```

Comment:
This was a nightmare to properly type hint. Basically, `find` and `findall` all return `Optional[Element]` because it may not find anything. But we know that the file we're processing will always return `Element`, so we need to tell it that it doesn't indeed work. We could instead add `if foo is not None` checks everywhere, but that would make this even more complicated.